x-pack/test_serverless/functional/test_suites/search/rules/rule_details.ts

```diff
@@ -138,7 +146,9 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
         await actionsMenuItemElem.at(1)?.click();
 
-        await retry.try(async () => {
+        await (await testSubjects.find('confirmModalConfirmButton')).click();
+
+        await retry.tryForTime(30 * 1000, async () => {
```

Comment:
Could we reduce it a bit as well? I understand this might need bit more time as we are waiting for rule to be disabled.